yotiprotoattr/List.proto

```diff
@@ -1,13 +1,13 @@
 syntax = "proto3";
 
-package Yoti.Protobuf.attrpubapi_v3;
+package attrpubapi_v1;
```

Comment:
Should this be `attrpubapi` instead if we are getting rid of the version?

Reply:
I'm just returning it to what it was before, and what is currently present on https://github.com/lampkicking/sdk-protobuf. 

I believe this line isn't actually used anyway, since we are overriding it with the specific go package name below